Components/Windows/create-local-user/README.md

```diff
@@ -0,0 +1,42 @@
+# Creation of a local Windows user account with RDP login permissions
+
+This is a sample component that demonstrates how to create a local Windows user account without Administrator permissions and grant RDP login privileges to this account. This can be useful in situation where you build a jump host that you do not want to join into an Active Directory and where you do not want the users to have admin access, especially if you manage your configuration changes to the machines via an EC2 Image Builder pipeline.
+
+This component activates the AWS Tools for Powershell for Windows [documentation](https://docs.aws.amazon.com/powershell/latest/userguide/pstools-welcome.html) which are already pre-installed on the AWS Managed AMIs for Windows Server 2019 and probably other versions as well. This is done so that the Powershell part of this component can interact with AWS Secrets Manager to retrieve the password for this users, as we did not want to have it written down in the document. To allow the build instance to retrieve the password for the user from AWS Secrets Manager the IAM role assigned to it via the EC2 Image Builder Infrastructure configuration needs to have a policy attached which allows the action ```secretsmanager:GetSecretValue``` for the specified secret.
+
+As the password is set during the runtime of the pipeline the instances deployed from the final AMI don't need access to this secret. 
+
+## Prerequisites
+
+- a Secret created in AWS Secrets Manager that stores the password for the local Windows user account
+- an IAM policy that allows the build instance to fetch the secret by allowing the ```secretsmanager:GetSecretValue``` API action
+- (if set) The resource policy on the secret needs to allow the build instance IAM role to read it
+- (if set) The resource policy on the KMS key protecting the secret needs to allow the build instance IAM role to use it for decryption
+- a Windows image build pipeline (This component was tested with the AWS provided Windows AMIs for Server 2019)
```

Comment:
It looks like the minimum requirement for this is Windows Server 2016, given you are using the ```Get-LocalUser``` cmdlet which doesn't exist on Windows Server 2012 R2.

Reply:
I tested it now on 3 versions and have adjusted this in the docs.